src/commands/StopCommand.ts

```diff
@@ -18,7 +18,7 @@ export class StopCommand extends BaseCommand {
         message.guild?.queue?.voiceChannel?.leave();
         message.guild!.queue = null;
 
-        message.channel.send(createEmbed("info", "⏹ Queue stopped..."))
+        message.channel.send(createEmbed("info", "⏹ The queue has been stopped."))
```

Comment:
```suggestion
        message.channel.send(createEmbed("info", "⏹ Queue stopped."))
```
Queue stopped is good, but delete the 2 dot